src/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesEntity.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { CollectionsHubLinkedCollections } from "Apps/__tests__/Fixtures/Collections"
+import { mount } from "enzyme"
+import React from "react"
+import { ArtistSeriesEntity, ArtworkImage } from "../ArtistSeriesEntity"
+jest.unmock("react-tracking")
+
+describe("ArtistSeriesEntity", () => {
+  let props
+
+  beforeEach(() => {
+    props = {
+      member: CollectionsHubLinkedCollections.linkedCollections[0].members[0],
+    }
+  })
+
+  it("Renders expected fields", () => {
+    const component = mount(<ArtistSeriesEntity {...props} />)
```

Comment:
It feels like there are a lot of unrelated assertions in this test. I think it would be beneficial to break them into individual tests, with names that tell us why the assertions are being made.

## Some examples of what I mean

- This test asserts that the text should _not_ match "Jasper Johns:". Why not? A separate test with a more specific name would tell me that. As this test stands, I don't know _why_ this is expected.

- We've rolled a test for the image width into this test, in addition to the [other test](https://github.com/artsy/reaction/pull/2644/files#diff-b458b45b0c5ac2ab639a1327179b482eR35). I'd rather see three individual tests to test the specific image widths - something like 
  1. `it("uses small image width when there are more than 2 hits")`
  2. `it("uses medium image width when there are 2 hits")`
  3. `it("uses large image width when there is 1 hit")`

- This test asserts that the `alt` text on the image equals `"Jasper Johns, Flag"`. I could see two separate tests for this:
  1. `it("uses the hit title for alt text if there is no artist")`, which would assert that the alt text was "Flag" if there was no associated artist
  2. `it("uses the artist name and hit title for alt text if there is an artist")`, which would assert what this test is asserting on line 31.

- There is another test case that is missing: `it("displays price guidance if it exists")`, which would verify that [this branch](https://github.com/artsy/reaction/pull/2644/files#diff-8a679cbe18e53c93fa1219c639cfbd82R49) is rendering when there is `price_guidance`.

## Why this matters

It's great to write tests for code coverage and to catch future regressions when we change the code, but it's important that they provide context to the person reading them when they break. That person is me, or someone else on the team, or even you a few months from now. They don't have the context that you have right now - adding smaller tests with more specific names preserves the context you have right now, and makes it easier for them to figure out _why_ a test broke.